src/tests/ftest/util/ior_test_base.py

```diff
@@ -232,19 +232,18 @@ def run_ior(self, manager, processes, intercept=None, display_space=True,
                 Default is self.pool.
             out_queue (queue, optional): Pass the exception to the queue.
                 Defaults to None.
-             env (EnvironmentVariables, optional): Environment to be used
+            env (EnvironmentVariables, optional): Environment to be used
              when running ior. Defaults to None
         """
         if not env:
             env = self.ior_cmd.get_default_env(str(manager), self.client_log)
-            if intercept:
-                env['LD_PRELOAD'] = intercept
-                env['D_LOG_MASK'] = 'INFO'
-                env['D_IL_REPORT'] = '1'
```

Comment:
Any specific reason 'D_IL_REPORT' is removed? Will it's removal impact other interception tests?

Reply:
Good catch, fixing it